libursa/src/cl/constants.rs

```diff
@@ -24,12 +24,15 @@ pub const LARGE_ALPHATILDE: usize = 2787;
 
 // Constants that are used throughout the CL signatures code, so avoiding recomputation.
 lazy_static! {
-    pub static ref LARGE_E_START_VALUE: BigNumber = BIGNUMBER_2.exp(
-                &BigNumber::from_u32(LARGE_E_START).unwrap(),
-                None).unwrap();
-    pub static ref LARGE_E_END_RANGE_VALUE: BigNumber = BIGNUMBER_2.exp(
-                &BigNumber::from_u32(LARGE_E_END_RANGE).unwrap(),
-                None).unwrap().add(&LARGE_E_START_VALUE).unwrap();
-    pub static ref LARGE_VPRIME_PRIME_VALUE: BigNumber = BIGNUMBER_2.exp(
-        &BigNumber::from_u32(LARGE_VPRIME_PRIME - 1).unwrap(), None).unwrap();
+    pub static ref LARGE_E_START_VALUE: BigNumber = BIGNUMBER_2
```

Comment:
Can you comment on what lazy_static! give us? Why use this macro instead of manually declaring them?

Reply:
The lazy aspect just means it won't create these values until they are used the first time. It's only for the CL signatures and proofs.